cfbs/commands.py

```diff
@@ -531,7 +532,12 @@ def update_command(to_update):
 
         if not update.version:
             update.version = index_info["version"]
+        print(
+            "Updated module '%s' from version %s to version %s"
+            % (update.name, old_version, update.version)
+        )
         updated.append(update)
+    print("Updated %d module%s" % (len(updated), "" if len(updated) == 1 else "s"))
```

Comment:
Why not gather these messages and then use them for both the commit message and standard output?